packages/react-native-editor/src/api-fetch-setup.js

```diff
@@ -43,10 +36,9 @@ const fetchHandler = ( { path }, retries = 20, retryCount = 1 ) => {
 };
 
 export const isPathSupported = ( path ) =>
-	[
-		/wp\/v2\/(media|categories)\/?\d*?.*/i,
-		/wpcom\/v2\/gutenberg\/.*/i,
-	].some( ( pattern ) => pattern.test( path ) );
+	[ /wp\/v2\/(media|categories)\/?\d*?.*/i ].some( ( pattern ) =>
+		pattern.test( path )
+	);
```

Comment:
Maybe we could add a comment here saying we should add only wp.org api paths here, just to cover our backs in the future.

Reply:
That's a good idea, will add that 👍 Thanks!